### PR TITLE
Add "api-supported" property

### LINKED ISSUE #1: Improve orientation non supported feedback
Hi,

When using this component, we may need to know if the orientation is supported or not by the device.

Maybe add another property to know if the orientation is available or not.

### PR DESCRIPTION
**Pull request by FluorescentHallucinogen**
Fix #1.

### CODE REVIEW

File: device-orientation.html [hunk 1]
```diff
@@ -32,15 +32,26 @@
...
           type: Object,
           notify: true
+        },
+
+        /**
+         * Indicates whether Device Orientation API is supported or not.
+         */
+        supported: {
+          type: Boolean,
+          value: false,
```
Comment (abdonrd) [inline device-orientation.html]: Please, remove the default value.
If the browser supports the Device Orientation API, we should never show `false`.

**Review by abdonrd — CHANGES_REQUESTED** [review]
Good job! 😄

File: device-orientation.html (hunk 1 above)
Comment (FluorescentHallucinogen) [reply device-orientation.html]: Done.

**Comment by FluorescentHallucinogen** [comment]
I've renamed "supported" property to "api-supported".

**Comment by FluorescentHallucinogen** [comment]
@abdonrd I've removed `value: false` from the code. Is Polymer property default value optional? Shouldn't I use `value: ''` instead of removing it?

**Comment by FluorescentHallucinogen** [comment]
@abdonrd Or should I use `value: null`?

**Comment by abdonrd** [comment]
@FluorescentHallucinogen yes, the default value is optional.

**Comment by abdonrd** [comment]
We should add the new property to the demo & inline-demo (README.md).

**Comment by FluorescentHallucinogen** [comment]
I'll do it right now. )

**Comment by FluorescentHallucinogen** [comment]
Done. ;)

**Review by abdonrd — APPROVED** [review]

**Comment by abdonrd** [comment]
Thanks @FluorescentHallucinogen! 👏

**Comment by FluorescentHallucinogen** [comment]
I think we should increment MINOR in MAJOR.MINOR.PATCH semantic version, bacause new functionality added in a backwards-compatible manner.

**Comment by abdonrd** [comment]
@FluorescentHallucinogen I'm thinking in first add #7.

**Comment by abdonrd** [comment]
@FluorescentHallucinogen done! [device-orientation v0.3.0](https://github.com/abdonrd/device-orientation/releases/tag/v0.3.0)

**Comment by FluorescentHallucinogen** [comment]
@abdonrd 
> done! device-orientation v0.3.0

Oh, I think `sensors-available` property should be added to the demo (`demo/index.html`) and inline-demo (`README.md`) too.

**Comment by FluorescentHallucinogen** [comment]
I'll add it now. ;)

**Comment by abdonrd** [comment]
It's already there. 😕

**Comment by FluorescentHallucinogen** [comment]
Oh, sorry, you are right. Good job! 😉

**Pull request merged**